htdocs/contrat/class/contrat.class.php

```diff
@@ -1157,7 +1158,8 @@ public function create($user)
 				return -2;
 			}
 		} else {
-			$this->error = $langs->trans("UnknownError: ".$this->db->error()." -", LOG_DEBUG);
+			$this->error = $langs->trans("UnknownError").": ".$this->db->error();
+			dol_syslog(get_class($this)."::create - 10 - ".$this->error, LOG_ERR);
```

Comment:
Note that when an error occurs after a sql request, it is already logged automatically with a dol_syslog.